app/src/main/java/nodomain/freeyourgadget/gadgetbridge/model/DeviceType.java

```diff
@@ -37,6 +37,7 @@
     LIVEVIEW(30, R.drawable.ic_device_default, R.drawable.ic_device_default_disabled),
     HPLUS(40, R.drawable.ic_device_hplus, R.drawable.ic_device_hplus_disabled),
     MAKIBESF68(41, R.drawable.ic_device_hplus, R.drawable.ic_device_hplus_disabled),
+    EXRIZUK8(40, R.drawable.ic_device_hplus, R.drawable.ic_device_hplus_disabled),
```

Comment:
I think you mean 42 here. The keys must be unique, otherwise DeviceType.fromKey() would not work.